src/FsAutoComplete.Core/AdaptiveExtensions.fs

```diff
@@ -531,53 +536,90 @@ module AsyncAVal =
   let ofTask (value: Task<'a>) = ConstantVal(value) :> asyncaval<_>
 
   let ofCancellableTask (value: CancellableTask<'a>) =
-    let mutable cache: Option<AdaptiveCancellableTask<'a>> = None
 
     { new AbstractVal<'a>() with
-        member x.Compute t =
-          if x.OutOfDate || Option.isNone cache then
-            let cts = new CancellationTokenSource()
-
-            let cancel () =
-              cts.TryCancel()
-              cts.TryDispose()
+        member x.Compute _ =
```

Comment:
Shouldn't be using a cache here since `CancellableTask` is a function, it should be re-run.